src/Kernel32.Shared/Kernel32+CreateFileFlags.cs

```diff
@@ -146,13 +145,13 @@ public enum CreateFileFlags : uint
             SECURITY_SQOS_PRESENT = 0x00100000,
 
             /// <summary>Impersonates a client at the Anonymous impersonation level.</summary>
-            SECURITY_ANONYMOUS = SECURITY_IMPERSONATION_LEVEL.SecurityAnonymous << 16,
+            SECURITY_ANONYMOUS = 0 << 16,
 
             /// <summary>The security tracking mode is dynamic. If this flag is not specified, the security tracking mode is static.</summary>
             SECURITY_CONTEXT_TRACKING = 0x00040000,
 
             /// <summary>Impersonates a client at the Delegation impersonation level.</summary>
-            SECURITY_DELEGATION = SECURITY_IMPERSONATION_LEVEL.SecurityDelegation << 16,
+            SECURITY_DELEGATION = 3 << 16,
```

Comment:
I think we have some decision to take here.

Header files are structured with `WINAPI_FAMILY_PARTITION` macros such that both the values and the original enum (`SECURITY_IMPERSONATION_LEVEL`) are accessible from store apps even if they can't actually occur (AFAIK, `SECURITY_IMPERSONATION_LEVEL` is documented desktop only).
I suspect that it's there for ease of porting code without having to ifdef every usage of the unsupported values.

Should we:
* Mirror the header files ? (And present to store apps values and potentially whole enums that can't be used)
* Use `#ifdef` to try to signal as much as possible what is or isn't available ?
* Do what I did there (maybe in a better way) importing values but not whole new types ?